cllc-public-app/Models.Extensions/Contact.cs

```diff
@@ -314,6 +323,14 @@ public static void CopyValuesNoEmailPhone(this MicrosoftDynamicsCRMcontact to, V
             to.AdoxioSelfdisclosure = from.SelfDisclosure;
             to.AdoxioIdentificationtype = (int?)from.PrimaryIdentificationType;
             to.AdoxioSecondaryidentificationtype = (int?)from.SecondaryIdentificationType;
+            to.AdoxioPhslivesincanada = (int?)from.PhsLivesInCanada;
```

Comment:
Do we also need to set AdoxioPhsconnectionsdetails on this routine and also in ToModel?